src/layouts/BaseLayout.astro

```diff
@@ -7,21 +7,23 @@ import ItemPageHeader from "@components/ItemPageHeader/index.astro";
 import { getCurrentLocale, useTranslations } from "@i18n/utils";
 import "@styles/base.scss";
 
-interface BaseLayoutProps {
+interface Props {
   title: string;
   className?: string;
-  variant: "root" | "item";
+  variant?: "root" | "item";
+  localize?: boolean;
```

Comment:
I don't love this solution but i think its useful for cases where we don't want/expect the title of the page to be localized. For example, the title of a sketch, or the name of a p5 method.